### ISSUE TITLE
Toggle command execution conformation

### ISSUE THREAD

**Issue by VorTechnix** [sbrl/Minetest-WorldEditAdditions#30]
I'd like to request the addition of a config/settings file to this mod. Further I'd like to request the option to turn off command execution conformation for large operations (requiring `//y`/`//n` when a large number of blocks will be effected) be added to the settings file.

**Comment by sbrl** [comment]
Hey there! Unfortunately, that's really only something that can be implemented in WorldEdit itself, and could be potentially rather dangerous. However, you can use the `//multi` command as a workaround like this:

```worldedit
//multi //set dirt //y
```

For `//subdivide` I've already implemented logic to bypass saferegion functionality (the internal name for the `//y` execution confirmation), so I could potentially implement an `//unsafe` command like this:

```worldedit
///unsafe set dirt
```

....but I'm wary of doing that, because the safe region functionality is there to avoid awkward issues, and I've rather avoid encouraging people to accidentally crash their machines :P

Note that for large regions, in the latest master `//subdivide` is now optimised for extremely large regions (>= 500 million nodes). It took some doing, but I'm rather pleased with the result (though servers with limited amounts of RAM tend to struggle at that scale).

**Comment by VorTechnix** [comment]
Sorry, didn't know it was a feature of the main mod... Forget I said anything.

**Comment by sbrl** [comment]
No worries, @VorTechnix! I'll close this issue for now, but I'm happy to reopen if there's anything further to discuss here.

**Comment by VorTechnix** [comment]
Now that we're implementing WEA Core I think we should look into this again. With core control this would cease to be an upstream issue.

**Comment by sbrl** [comment]
Absolutely. given we have our own `safe_region` implementation now, this should not be that much of a problem.

We have yet to implement a storage interface for persisting things though, so that part may have to wait a bit longer.
